modules/ROOT/pages/prepare-raml-task.adoc

```diff
@@ -26,15 +27,35 @@ And then you can apply this trait in each individual operation like this:
     description: Gets a list of all the inventory products.
 ----
 
-. Click Policies on the API dashboard.
+== Getting Policies Specification Snippets
+
+. Click *Policies* on the API dashboard.
 +
-In the list of policies, the RAML snippet link contains the RAML code you need to add to the RAML.
+In the list of policies, the RAML snippet link contains the RAML trait or OAS components you need to add to the API specification.
 +
 image::raml-snippet.png[]
-. Click RAML Snippet.
+. Click *API Specification Snippet*.
+. Select the API type to match your specification:
+** RAML 0.8 or RAML 1.0 to match the API RAML version.
 +
 image::snippet-rate-limiting.png[]
-. Select RAML 0.8 or RAML 1.0 to match the API RAML version.
+** Or OAS 2.0 to match the API OAS version.
++
+image::prepare-raml-task-2b676.png[]
++
+OAS Snippets is not supported by all policies versions. +
```

Comment:
@fermujica more than supported is actually displayed/shown.

Reply:
👍 
Tell me if this works better: 92768f0